tests/phpunit/Unit/Graph/GraphTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SRF\Tests\Unit\Formats;
+
+use SMW\Test\QueryPrinterRegistryTestCase;
+use SRF\Graph\GraphNode;
+
+
+/**
+ * Tests for the SRF\Graph class.
```

Comment:
Wrong

Instead: `@covers SRF\Graph\GraphPrinter`

Reply:
Hold on... You are actually testing the GraphNode class... Then why are you extending `QueryPrinterRegistryTestCase` and implementing a bunch of stuff you don't need?

Reply:
see: https://github.com/RacoonDev/SemanticResultFormats/commit/c1acdf0cf7f136a108ae0380959954703d0bbd35